packages/core/src/connectors/add-connectors.ts

```diff
@@ -0,0 +1,72 @@
+import { exec } from 'child_process';
+import { existsSync } from 'fs';
+import { mkdir, rename, unlink } from 'fs/promises';
+import path from 'path';
+import { promisify } from 'util';
+
+import chalk from 'chalk';
+import got from 'got';
+import rimraf from 'rimraf';
+import tar from 'tar';
+
+import { npmPackResultGuard } from './types';
+
+const execPromise = promisify(exec);
+
+const fetchConnectorsList = async () => {
+  // Will change to "logto-io/connectors" once the new repo is ready.
+  const directories = await got
+    .get('https://api.github.com/repos/logto-io/logto/contents/packages')
+    .json<Array<{ name: string }>>();
+
+  return (
+    directories
+      // Will be removed once the new repo is ready.
+      .filter(
+        ({ name }) =>
+          name.startsWith('connector-') &&
+          name !== 'connector-core' &&
+          name !== 'connector-sendgrid-mail'
```

Comment:
curious why treat SendGrid-mail as a special case.

Reply:
The package name in `package.json` is "connector-sendgrid-email", not the same as the folder name.

Reply:
Any particular reason for this? Can't we use the same name?

Reply:
maybe we can simply rename the folder to fix this

Reply:
The folder name is renamed in the new repo: https://github.com/logto-io/connectors/tree/master/packages/connector-sendgrid-email